packages/core/src/app/shipping/ShippingHeader.tsx

```diff
@@ -36,7 +48,25 @@ const ShippingHeader: FunctionComponent<ShippingHeaderProps> = ({
                     />
                 </Legend>
 
-                {shouldShowMultiShipping && (
+                {showConfirmationModal && (
+                    <>
+                        <ConfirmationModal
+                            action={handleShipToSingleConfirmation}
+                            headerId="shipping.ship_to_single_action"
+                            isModalOpen={isModalOpen}
+                            messageId="shipping.ship_to_single_message"
+                            onRequestClose={() => setIsModalOpen(false)}
+                        />
+                        <a
+                            data-test="shipping-mode-toggle"
+                            href="#"
+                            onClick={preventDefault(() => setIsModalOpen(true))}
+                        >
```

Comment:
The code indentation is missing.

Reply:
`<ConfirmationModal>` and `<a>` tags are siblings here, so should be at same ident.

Reply:
🤦🏻‍♂️Oh mine....Yes, I missed the closing `</a>`.